token/ts/package.json

```diff
@@ -6,19 +6,34 @@
     "repository": "https://github.com/solana-labs/solana-program-library",
     "license": "Apache-2.0",
     "engines": {
-        "node": ">= 10"
+        "node": ">= 14"
     },
     "publishConfig": {
         "access": "public"
     },
+    "files": [
+        "lib",
+        "src",
+        "README.md"
+    ],
     "type": "module",
     "sideEffects": false,
-    "main": "lib/index.js",
+    "main": "lib/index.cjs.js",
+    "module": "lib/index.esm.js",
     "esnext": "lib/index.js",
     "types": "lib/index.d.ts",
+    "browser": {
+        "./lib/index.cjs.js": "./lib/index.browser.cjs.js",
+        "./lib/index.esm.js": "./lib/index.browser.esm.js"
+    },
+    "browserslist": [
```

Comment:
As far as I know, and as sad as it is, `rollup` & `@rollup/plugin-typescript` do not take `browserslist` into consideration. `babel` with `@babel/preset-env` do.

You can double check by bundling with and without this configuration and verify that the build outputs are the same.

Reply:
Ah, good point. I just lifted that from somewhere (probably web3.js). How much does it matter, is it worth adding babel?

Reply:
It depends. It's a trade-off between complexity leading to increased maintenance cost over time & output quality, especially in regard to transforming & (not) polyfill-ing the output as close as possible to the defined browser targets.

Some TS features are also not supported by [`@babel/plugin-transform-typescript`](https://babeljs.io/docs/en/babel-plugin-transform-typescript/)